tests/foreman/ui/test_reporttemplates.py

```diff
@@ -393,6 +393,42 @@ def test_positive_schedule_generation_and_get_mail(session):
                       The result is compressed.
     :CaseImportance: High
     """
+    # make sure we have some subscriptions
+    with manifests.clone() as manifest:
+        upload_manifest(module_org.id, manifest.content)
+    # generate Subscriptions report
+    with session:
+        session.reporttemplate.schedule(
+                "Subscriptions", values={'output_format': 'JSON', 'generate_at': '1970-01-01 17:10:00', 'email': True, 'email_to': 'root@localhost'}
+        )
+    file_path = '/tmp/{0}.json'.format(gen_string('alpha'))
+    gzip_path = '{0}.gz'.format(file_path)
+    expect_script = '''#!/usr/bin/env expect
+spawn mail
+expect "& "
+send "w $ /dev/null\\r"
+expect "Enter filename"
+send "\\r"
+expect "Enter filename"
+send "\\r"
+expect "Enter filename"
+send "\\025{0}\\r"
+expect "&"
+send "q\\r"
+'''.format(gzip_path)
+    ssh.command('expect -c \'{0}\''.format(expect_script), hostname=settings.server.hostname)
+    #ssh.command('gunzip {0}'.format(gzip_path), hostname=settings.server.hostname)
+    #ssh.download_file(file_path)
+    ssh.download_file(gzip_path)
+    os.system('gunzip {0}'.format(gzip_path))
```

Comment:
Here, and in other places, it would probably be more readable to use f-strings rather than `.format()`. But up to you.